Drop python3 config dir from library search list to avoid linking against the
static library that may exist there (bug #669306).

diff --git a/src/configure.ac b/src/configure.ac
index a098da82e..7aedd7bee 100644
--- a/src/configure.ac
+++ b/src/configure.ac
@@ -1496,7 +1496,7 @@ eof
 	    dnl -- delete the lines from make about Entering/Leaving directory
 	    eval "`cd ${PYTHON3_CONFDIR} && make -f "${tmp_mkf}" __ | sed '/ directory /d'`"
 	    rm -f -- "${tmp_mkf}"
-	    vi_cv_path_python3_plibs="-L${PYTHON3_CONFDIR} -lpython${vi_cv_var_python3_version}${vi_cv_var_python3_abiflags}"
+	    vi_cv_path_python3_plibs="-lpython${vi_cv_var_python3_version}${vi_cv_var_python3_abiflags}"
 	    vi_cv_path_python3_plibs="${vi_cv_path_python3_plibs} ${python3_BASEMODLIBS} ${python3_LIBS} ${python3_SYSLIBS}"
 	    dnl remove -ltermcap, it can conflict with an earlier -lncurses
 	    vi_cv_path_python3_plibs=`echo $vi_cv_path_python3_plibs | sed s/-ltermcap//`
